### ISSUE TITLE
Profiles can't be edited due to 'Buckingham'

### ISSUE THREAD

**Issue by pmk01** [DemocracyClub/yournextrepresentative#1435]
We've had reports of people being unable to edit profiles due to 'Buckingham' appearing in the add ned ballot drop down. Has been reported on:

https://candidates.democracyclub.org.uk/person/74706/peter-schwier
https://candidates.democracyclub.org.uk/person/75033/nigel-mercer

I can't replicate it but see photo from user

![vfpHYys0SRkv5Uc8FaVaQXpmIWidy9YdkA](https://user-images.githubusercontent.com/61510609/114170368-c48a3480-992a-11eb-84b5-2eff1ae939d1.jpeg)

**Comment by pmk01** [comment]
From one user:
the system is Mac OS Mojave
Browser is Safari

**Comment by michaeljcollinsuk** [comment]
@symroe @pmk01 Just to confirm I am able to recreate this is Safari - going to investigate

**Comment by pmk01** [comment]
This from another user:

Yes I attempted to use Firefox and Safari. Mac OS version 10.15.7.

I didn’t take a screen grab, sorry. 

Just below the correct poll details ‘Yarm Ward Stockton on Tees’ the was a second poll.

I think the area was Birmingham and there was a drop down to add a political party.

On clicking Save (or whatever the button said) the error stated that I needed to add a party to continue. This was not possible as ‘Independent’ was not an option. I did manage to change the location to ‘Yarm Ward Stockton on Tees’, but I could still not add a party.

That was it. I noticed that some of the details updated, but not the image or the Candidate Statement.
